content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -364,16 +380,8 @@ The following configuration variables are available for both automatic and custo
 
 `DD_TRACE_AGENT_URL`
 : **TracerSettings property**: `Exporter.AgentUri`<br>
-Sets the URL endpoint where traces are sent. Overrides `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` if set. <br>
-**Default**: `http://<DD_AGENT_HOST>:<DD_TRACE_AGENT_PORT>`
-
-`DD_AGENT_HOST`
-: Sets the host where traces are sent (the host running the Agent). Can be a hostname or an IP address. Ignored if `DD_TRACE_AGENT_URL` is set. <br>
-**Default**: `localhost`
-
-`DD_TRACE_AGENT_PORT`
-: Sets the port where traces are sent (the port where the Agent is listening for connections). Ignored if `DD_TRACE_AGENT_URL` is set. <br>
-**Default**: `8126`
```

Comment:
I don't think we should unlist these completely. That implies to me that they do anything. However, as we've seen in https://github.com/DataDog/dd-trace-dotnet/issues/2426, they absolutely _do_ have an impact, and can cause invalid configurations.

We have a deprecated settings section - maybe we should move them there?

Reply:
Yes, I've also raised the point to Yoann to see how we should treat it. Because it's documented approximatively everywhere, so considering it deprecated may be just my pov.

Reply:
Ok reintroduced in latest commit